lib/electric_slide/call_queue.rb

```diff
@@ -292,7 +292,7 @@ def bridge_agent(agent, queued_call)
 
       agent.callback :connect, self, agent.call, queued_call
 
-      agent.join queued_call
+      agent.join queued_call if queued_call.active?
     rescue Celluloid::DeadActorError, Adhearsion::Call::Hangup, Adhearsion::Call::ExpiredError, Adhearsion::Call::CommandTimeout
```

Comment:
You should rescue Punchblock::ProtocolError here since it's `agent.join` that will raise it.